packages/zowe-explorer/src/webviews/package.json

```
@@ -11,18 +11,22 @@
     "preview": "vite preview",
     "fresh-clone": "pnpm clean && rimraf node_modules",
     "clean": "rimraf dist || true",
-    "package": "echo \"webviews: nothing to package.\"",
+    "package": "node -e \"fs.accessSync(path.join(__dirname, 'dist'))\" && echo \"webviews: nothing to package.\" || pnpm build",
```

No change required, but we may be able to use a relative path here since I think npm scripts already run in the context of the current directory 😋 
```suggestion
    "package": "node -e \"fs.accessSync('dist')\" && echo \"webviews: nothing to package.\" || pnpm build",
```